iina/PlayerCore.swift

```diff
@@ -698,6 +728,15 @@ class PlayerCore: NSObject {
     // querying `mainWindow.isWindowLoaded` will initialize mainWindow unexpectly
     guard let mw = mainWindow, mw.isWindowLoaded else { return }
 
+    if info.disableOSDForFileLoading {
+      switch osd {
```

Comment:
No, I mean that you have a `switch` that `break`s and a `default` that `returns`. Why not use
```swift
guard osd == .fileStart else {
	return
}
```